### DISCUSSION TITLE
GitHub Organizations

### DISCUSSION THREAD

**Discussion by bhagyesh-metacube** [community/community#54430] [Enterprise]
Hi Team,

We are planning to purchase the GitHub Pro license for our company but need to discuss about GitHub Organizations feature to clarify our doubts.

Please let me know if we can connect over a quick call.

Thanks,
Bhagyesh Jain
Metacube Software Pvt. Ltd.

**Comment by queenofcorgis** [comment]
Hi there 👋🏼 

If you'd like to connect with our team, please use this link: https://github.com/enterprise/contact. 

Thank you!
